src/react/components/pages/export/exportForm.test.tsx

```diff
@@ -73,7 +95,7 @@ describe("Export Form Component", () => {
         };
 
         const props: IExportFormProps = {
-            settings: null,
+            settings: defaultExportSettings,
```

Comment:
FYI - the settings is not "required" on the form.  I would verify or add additional tests that this is still the case.

Reply:
Ditto other test comment